### PR TITLE
Align DesignSystem Colors

### PR DESCRIPTION
**Pull request by jkmassel**
<figure>
Screenshot 2024-09-04 at 12 54 07 PM
<figcaption>Left side is new, right side is original. It's substantially the same, but the old setup relied on being the source of truth for what a "brand" color is.
</figcaption >
</figure>

Building on #23528, this PR removes the `DS.*` APIs – everything should now be under `AppColor`, or should use a system default.

It also fixes the Tab Bar selected state, and the "Back" button color

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/Utility/App Configuration/AppColor.swift [hunk 1]
```diff
@@ -86,10 +87,12 @@ struct AppColor {
 
     #if IS_JETPACK
     static let brand = CSColor.JetpackGreen.base
+    static let _brand = Color(CSColor.JetpackGreen.base)
     #endif
 
     #if IS_WORDPRESS
     static let brand = CSColor.WordPressBlue.base
+    static let _brand = Color(CSColor.WordPressBlue.base)
```
Comment (kean) [inline WordPress/Classes/Utility/App Configuration/AppColor.swift]: Add `AppColor` and `UIAppColor`?

**Review by kean — APPROVED** [review]
Looks good, but I added one comment, and also the contrast of "tertiary disabled" is too low.

File: WordPress/Classes/Utility/App Configuration/AppColor.swift (hunk 1 above)
Comment (jkmassel) [reply WordPress/Classes/Utility/App Configuration/AppColor.swift]: Sure – done in c7d3a4e82ba75aba0ddee2f1d7f4d6a7f242ac54

**Comment by jkmassel** [comment]
The tertiary styles will always be a bit tricky if the base color isn't saturated enough – I removed the transparency and put a better tint color on the gallery in fc41eae35c072eb18fee5cbb1b0915ff79700552

**Pull request merged**